plugin/web/sharedComponents/ModifiedImage/hooks.ts

```diff
@@ -0,0 +1,77 @@
+import { useState, useEffect } from "react";
+import tinycolor from "tinycolor2";
+
+type Props = {
+  imageUrl: string;
+  blendColor: string;
+  width: number;
+  height: number;
+};
+
+const useModifiedImage = ({ imageUrl, blendColor, width, height }: Props) => {
+  const [modifiedImageUrl, setModifiedImageUrl] = useState("");
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const image = new Image();
+    image.crossOrigin = "anonymous";
+    image.src = imageUrl;
+    image.onload = () => {
+      if (!image.complete) {
+        console.error("Failed to load image due to cross-origin error");
```

Comment:
Another cross origin error that might be better more generic

Reply:
done